FIRBoundaries.dat

```diff
@@ -15505,7 +15855,7 @@ RJCG|0|0|11|21.000000|125.416667|50.083333|165.000000|41.614100|138.370000
 39.227330|143.252500
 39.414300|141.910300
 39.165800|139.921700
-RJBG|0|0|7|21.004444|121.498611|30.003611|136.997500|25.504028|129.248056

```

Comment:
Why the removal of this when added just some time ago to fix with the new callsign

Reply:
My bad, used not up-to-date files